frontend/components/playlists/playlist_show_container.js

```diff
@@ -36,28 +40,29 @@ const mapStateToProps = ({
                 title: "Add to playlist",
                 submenu: [
                     [
-                        { 
-                            title: "Create new playlist"
+                        {
+                            title: "Create new playlist",
                         },
                         ...playlists,
-                    // Enclose array of playlists in an array since dropdown uses recursive .map function on items prop
-                    ]
+                        // Enclose array of playlists in an array since dropdown uses recursive .map function on items prop
```

Comment:
Consider writing long comments on two lines, this helps in reviews where it wraps around in the diff view

Reply:
Great to know, ty